dbms/programs/server/Server.cpp

```diff
@@ -864,7 +864,11 @@ int Server::main(const std::vector<std::string> & /*args*/)
         for (auto & server : servers)
             server->start();
 
-        setTextLog(global_context->getTextLog());
+        {
+            String level_str = config().getString("text_log.level", "");
+            int level = level_str.empty() ? 0 : Poco::Logger::parseLevel(level_str);
```

Comment:
Ok, but zero is "none" and we expect maximum logging to text log by default.

Reply:
Ok, thanks! (I was looking here and did not find that zero is used, but obviously I had not though about none )